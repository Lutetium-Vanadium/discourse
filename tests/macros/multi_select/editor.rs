fn main() {
    let q = requestty::questions![MultiSelect { editor: todo!() }];
}
