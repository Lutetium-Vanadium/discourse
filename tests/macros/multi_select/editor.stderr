error: option `editor` does not exist for kind `multi_select`
 --> tests/macros/multi_select/editor.rs:2:49
  |
2 |     let q = requestty::questions![MultiSelect { editor: todo!() }];
  |                                                 ^^^^^^
